elasticsearch_collectd.py

```diff
@@ -777,17 +777,19 @@ def sanatize_intervals(self):
                                       (self.index_interval %
                                        self.collection_interval)
                 log.warning('The Elasticsearch Index Interval must be \
-    greater or equal to than and divisible by the collection Interval.  The \
-    Elasticsearch Index Interval has been rounded to: %s' %
+                            greater or equal to than and divisible by the \
```

Comment:
This is going to create big gaps in the log text since it won't strip out the space at the beginning of the line.  Python has a trick where you can concat strings by just putting them on newlines with nothing between them.  See https://docs.python.org/2/reference/lexical_analysis.html#string-literal-concatenation